docs/WebUI.md

```diff
@@ -0,0 +1,73 @@
+# WebUI
+
+NNI is a research platform for metalearning. It provides easy-to-use interface so that you could perform neural architecture search, hyperparameter optimization and optimizer design for your own problems and models.
```

Comment:
The description of NNI should have already been somewhere else, we do not need to repeat it.